server/src/com/cloud/network/vpn/RemoteAccessVpnManagerImpl.java

```diff
@@ -596,7 +597,7 @@ public void doInTransactionWithoutResult(TransactionStatus status) {
 
 
         sb.and("id", sb.entity().getId(), SearchCriteria.Op.EQ);
-        sb.and("username", sb.entity().getUsername(), SearchCriteria.Op.EQ);
+        sb.and("username", sb.entity().getUsername(), SearchCriteria.Op.LIKE);
```

Comment:
I vote EQ instead LIKE here.
you may add new rules here in the next part, for example.

+        if (keyword != null) {
+            final SearchCriteria<VpnUserVO> ssc = _vpnUsersDao.createSearchCriteria();
+            ssc.addOr("username", SearchCriteria.Op.LIKE, "%" + keyword + "%");
+            ssc.addOr("state", SearchCriteria.Op.EQ, "%" + keyword + "%");
+            sc.addAnd("username", SearchCriteria.Op.SC, ssc);
+        }